torch/nn/functional.py

```diff
@@ -2316,6 +2316,63 @@ def poisson_nll_loss(input, target, log_input=True, full=False, size_average=Non
     return ret
 
 
+def gaussian_nll_loss(input, target, var, eps=1e-8, full=True, reduction='mean'):
```

Comment:
nit: this might be better to make these kwargs-only to ensure people use the api properly by doing:
```
 def gaussian_nll_loss(input, target, var, *, eps=1e-8, full=True, reduction='mean'): 
```

Same for the nn.Module version.

Reply:
Good idea, thanks.